src/minimap.h

```diff
@@ -0,0 +1,168 @@
+/*
+Minetest
+Copyright (C) 2010-2015 celeron55, Perttu Ahola <celeron55@gmail.com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#ifndef MINIMAP_HEADER
+#define MINIMAP_HEADER
+
+#include "irrlichttypes_extrabloated.h"
+#include "client.h"
+#include "voxel.h"
+#include "jthread/jmutex.h"
+#include <map>
+#include <string>
+#include <vector>
+
+enum MinimapMode {
+	MINIMAP_OFF,
```

Comment:
Could you prefix these enum entries so it's immediately apparent what types of values they are? e.g. MINIMAP_MODE_OFF, MINIMAP_MODE_SURFACE_x1, ...